src/kansa/components/Exhibition.jsx

```diff
@@ -0,0 +1,548 @@
+import React from 'react';
+
+import Checkbox from 'material-ui/Checkbox';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton'
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+import TextField from 'material-ui/TextField';
+const { Col, Row } = require('react-flexbox-grid');
+import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card'
+import Divider from 'material-ui/Divider';
+import Paper from 'material-ui/Paper';
+import FileInput from 'react-file-input';
+
+import BasicRules from './basic-rules';
+
+import { API_ROOT } from '../../constants'
+import API from '../../lib/api'
+
+const raami = new API(`${API_ROOT}raami/`);
+var ID = 0
+
+export default class ExhibitReg extends React.Component {
+
+  constructor(props) {
+    super(props);
+    
+    const member = props.params.id
+
+    this.state = {
+      id: ID,
+      person_id: parseInt(member),
+      name: '',
+      url:'',
+      description:'',
+      transport:'',
+      continent: '',
+      filename:'',
+      filedata: null,
+      legal: false,
+      auction: 0,
+      print: 0,
+      digital: false,
+      agent:'',
+      contact:'',
+      waitlist: false,
+      postage: 0,
+      open: false,
+      Works: [{ id: null, 
+            artist_id: ID, 
+            title: '',
+            width: 0,
+            height: 0,
+            depth: 0, 
+            technique: '', 
+            orientation: '', 
+            filename: '', 
+            filedata: null,
+            year: 0, 
+            price: 0, 
+            gallery: ''}
+            ]
+
+      }
+
+
+    raami.GET(`people/${member}`).then((data)=>{
+      if(data.length > 0 && data[0].id > 0) {
+        this.setState(data[0])
+        ID = data[0].id
+        var _work = this.state.Works.slice();
+        _work[0].artist_id = ID
+        raami.GET(`works/${ID}`).then(res => {
+          if('works' in res) {
+            res.works.forEach((item) => {
+              _work.unshift(item)
+              
+            })
+          this.setState({Works:_work})
+          }
+          console.log('artist',this.state)
+      })
+
+        }
+        
+      })
```

Comment:
Fix indentation. Also, should this and `this.setState({ person_id: parseInt(member) })` be also run in [componentWillReceiveProps](https://facebook.github.io/react/docs/react-component.html#componentwillreceiveprops)?